deps/wazuh_testing/wazuh_testing/tools/scans/dependencies.py

```diff
@@ -63,6 +63,8 @@ def prepare_input(pip_mode, input_file_path):
         with open(input_file_path, mode='r') as input_file:
             lines = input_file.readlines()
             for line in lines:
+                if line.startswith('\n') or line.startswith('#'):
```

Comment:
Using this approach avoid corner case of space and `#`, avoiding to explicitly specify `\n`

```suggestion
     if line.strip().startswith('#') or not line.strip():
```